jquery.flot.logaxis.js

```diff
@@ -88,9 +90,9 @@ Set axis.mode to "log" to enable.
             }
         });
 
-        if (maxIdx - minIdx <= noTicks / 4 && PREFERRED_LOG_TICK_VALUES.length < 700) {
+        if (maxIdx - minIdx <= noTicks / 4 && logTickValues.length !== EXTENDED_LOG_TICK_VALUES.length) {
             //try with multiple of 5 for tick values
-            PREFERRED_LOG_TICK_VALUES = computePreferedLogTickValues(Number.MAX_VALUE, 4);
+            logTickValues = EXTENDED_LOG_TICK_VALUES;
```

Comment:
I'd still like to see the powers of then showed preferentially. With this code you have no guarantee that something like 20, 50, 200, 500, 2000, 5000 will be generated when 20, 100, 200, 1000, 2000 would be preferable.

I'd like to something like subdividing the interval between the powers of ten to obtain new ticks. 

But maybe I'm wrong and the code does that.